src/olympia/addons/buttons.py

```diff
@@ -17,20 +17,22 @@ def install_button(context, addon, version=None,
     """
     If version isn't given, we use the latest version.
     """
+    if not collection:
+        collection = None
```

Comment:
this was to fix a weird error where the undefined `collection` variable in the jinja2 template was undefined but not "undefined" enough to not break `hasattr`/`getattr`